mlir/test/Conversion/MIGraphXToTosa/migraphx-to-rock-conv-nhwc.mlir

```diff
@@ -6,7 +6,7 @@
 // CHECK-SAME: memref<1x3x3x7xf32>
 func.func @convNHWC(%in: !migraphx.shaped<1x4x5x5xf32, 100x1x20x4>, %fil: !migraphx.shaped<7x4x3x3xf32, 36x1x12x4>) -> !migraphx.shaped<1x7x3x3xf32, 63x1x21x7> attributes {kernel, arch = "gfx1100", num_cu = 48 : i64} {
   // CHECK: rock.conv2d
-  // CHECK-SAME: filter_layout = ["k", "y", "x", "c", "g"]
+  // CHECK-SAME: filter_layout = ["k", "g", "y", "x", "c"]
```

Comment:
Does it matter? ROCK can support any layouts.

Reply:
Yes, yes it does, because that layout string represents the in-memory layout (as best it can)

More importantly, this matters because - I think - it means you've gotten your transformations backwards